### PR TITLE
OV-403: Create voices page

### LINKED ISSUE #403: FEAT: Create voices page
### What feature?

Create a page with list of available voices.

### Add screenshots

Image

### Acceptance criteria

_No response_

### PR DESCRIPTION
**Pull request by Alblupynos**
Create voices page FE part.
image

### CODE REVIEW

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/bundles/common/components/audio-player/audio-player.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const AudioPlayer: React.FC<Properties> = ({
         getAudioData(audioUrl)
             .then(({ durationInSeconds }) => {
                 setDurationInFrames(Math.round(durationInSeconds * FPS));
-                onSetDuration(durationInSeconds);
+                onSetDuration && onSetDuration(durationInSeconds);
```
Comment (lfelix3011) [inline frontend/src/bundles/common/components/audio-player/audio-player.tsx]: Could also use optional chaining operator onSetDuration?.(durationInSeconds);

Comment (sofiia-trokhymchuk) [inline frontend/src/bundles/common/components/audio-player/audio-player.tsx]: It is better to use if statement in this case

File: frontend/src/bundles/voices/components/voice-card/voice-card.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    voice: Voice;
+};
+
+const VoiceCard: React.FC<Properties> = ({ voice }) => {
+    const [url, setUrl] = useState<string | null>(null);
+    const [isLoading, setIsLoading] = useState(false);
+    const dispatch = useAppDispatch();
+
+    const text =
+        'Hello, I can handle video speech for you, choose me if you like it!';
```
Comment (sofiia-trokhymchuk) [inline frontend/src/bundles/voices/components/voice-card/voice-card.tsx]: Declare this const outside of the component

File: frontend/src/bundles/voices/components/voice-section/voice-section.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    color="background.600"
+                    bg="#D1D4DB"
+                    fontWeight="400"
+                    padding="2px 10px"
+                >
+                    {voices.length}
+                </Badge>
+            </Flex>
+
+            {voices.length > 0 ? (
```
Comment (sofiia-trokhymchuk) [inline frontend/src/bundles/voices/components/voice-section/voice-section.tsx]: Use const EMPTY_LENGTH

**Review by sofiia-trokhymchuk — CHANGES_REQUESTED** [review]

**Comment by sofiia-trokhymchuk** [comment]
Merge conflicts

File: frontend/src/bundles/common/components/sidebar/sidebar.tsx [hunk 4]
```diff
@@ -105,6 +106,23 @@ const Sidebar: React.FC<Properties> = ({ children }) => {
                             label="My Avatar"
                         />
                     </Link>
+                    <Text color="background.600" variant="bodySmall">
```
Comment (JKaypa) [inline frontend/src/bundles/common/components/sidebar/sidebar.tsx]: With this fontsize the text doesn't fit when sidebar is collaped

Comment (Alblupynos) [reply frontend/src/bundles/common/components/sidebar/sidebar.tsx]: Fixed

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**